zabbix/provider.go

```diff
@@ -16,7 +16,7 @@ import (
 func Provider() *schema.Provider {
 	p := &schema.Provider{
 		Schema: map[string]*schema.Schema{
-			"user": &schema.Schema{
+			"username": &schema.Schema{
```

Comment:
Not sure it is required to rename the provider attribute, as it is a breaking change.
Even so less if we want to maintain compatibility with older Zabbix versions which use `user`.